CHANGELOG.md

```diff
@@ -61,6 +61,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - `block` objective - added argument `noSafety` which disables removing progress when the player does the opposite of what the objective asks for
 - `hunger` condition and event
 - `variable` condition - forceSync argument forces the condition to be checked on the main thread 
+- holograms
+    - variable support
+    - top lists ordered by point values of players
```

Comment:
Not obvious, and i will change the changelog format a bit sonn, but this should be better placed between line 40 and 41